config/routes.rb

```diff
@@ -6,4 +6,6 @@
   get "/about", to: "static_pages#about"
   get "/contact", to: "static_pages#contact"
   get "/signup", to: "users#new"
+
+  resources :users
```

Comment:
resources dùng để làm gì?

Reply:
Daj, resources có thể tạo, show, sửa, xóa model ứng với 4 operation cơ bản của HTTP là POST, GET, PATCH, and DELETE ạ.